src/main/java/com/training/tinyurl/security/service/UserDetailsServiceImpl.java

```diff
@@ -18,10 +18,9 @@ public class UserDetailsServiceImpl implements UserDetailsService {
 
     @Override
     public UserDetails loadUserByUsername(String email) throws UsernameNotFoundException {
-        Optional<UserInfoEntity> user = userInfoDb.findById(email);
-        if(user.isPresent()){
-            return new AppUserDetails(user.get());
-        }
-        throw new UsernameNotFoundException("No record found for "+email);
+        String finalEmail = email.toUpperCase();
+        Optional<UserInfoEntity> user = userInfoDb.findById(finalEmail);
+        return user.map(AppUserDetails::new).
```

Comment:
There is no log present for "User login was unauthorized. No record found for ..".